### ISSUE TITLE
unable to make buildoss

### ISSUE THREAD

**Issue by vtolstov** [cockroachdb/cockroach#31655]
go get -d github.com/cockroachdb/cockroach and after that run make buildoss

host is fedora 28
```
build/node-run.sh -C pkg/ui ./node_modules/.bin/webpack --progress -p --config webpack.protos.js --env.dist=oss
Hash: 4e66ad26dbfe6374864e
Version: webpack 3.8.1
Time: 148ms
            Asset       Size  Chunks             Chunk Names
protos.oss.dll.js  530 bytes       0  [emitted]  protos
   [0] dll protos 12 bytes {0} [built]

ERROR in dll protos
Module not found: Error: Can't resolve './src/js/protos' in '/root/.cache/go-path/src/github.com/cockroachdb/cockroach/pkg/ui'
 @ dll protos
make: *** [Makefile:1152: pkg/ui/dist/protos.oss.dll.js] Error 2
make: *** Deleting file 'pkg/ui/dist/protos.oss.dll.js'
make: *** [pkg/ui/dist/protos.oss.dll.js] Deleting file 'pkg/ui/protos.oss.manifest.json'
```

**Comment by knz** [comment]
can you try this workaround:

1. run `make build` first
2. then `make buildoss`

does this work?

**Comment by vtolstov** [comment]
@knz in this case it works.

**Comment by benesch** [comment]
Hmm, I'm afraid I can't reproduce. Can you post the full output from the failed Make run? I'm also curious what version of Make you're running (`make --version`).

**Comment by benesch** [comment]
Never mind! I tried a little harder and found the typo. Thanks for the report. Fix landing soon; follow #31684.

**Comment by vtolstov** [comment]
Thanks!
